src/app/components/category-card/category-card.component.html

```diff
@@ -0,0 +1,91 @@
+<div class="card" style="margin: auto">
+    <form [formGroup]="categoryForm">
+      <div class="loading-indicator" *ngIf="loading === true">
```

Comment:
This should be outside of the form. The form should only contain form controls and submit/reset butons